apps/dashboard/app/views/projects/_form.html.erb

```diff
@@ -17,9 +17,10 @@
 %>
 
 <div class='card'>
-<h5 class='card-header'>Project Details</h5>
+<h4 class='card-header'>Project Details</h4>
```

Comment:
Oh I didn't realize this was an actual header tag. If this is going to be an actual header tag, it likely has to be an `h2` for accessibility reasons. There's an h1 on the page, and having an h4 only makes sense if there is an h2 and h3 above it.

I'd say make this an h2 directly or just style it to be large, but it shouldn't be an h3 or above.

https://www.w3.org/WAI/tutorials/page-structure/headings/

Reply:
Thank you for this. css is not my strong suit and I have always assumed that you pick the header that looks best. Subjective, I know...I am going to look for some articles on accessibility and familiarize myself with best practices.